tests/windows/conftest.py

```diff
@@ -3,8 +3,8 @@
 import pytest
 
 
-if sys.platform in ("win32", "cygwin"):
-    import msvcrt
+# if sys.platform in ("win32", "cygwin"):
+#     import msvcrt
```

Comment:
don't just comment out stuff and leaf dead code in the file. If its no longer needed, remove it.
```suggestion
```